src/rhsm/logutil.py

```diff
@@ -16,6 +16,15 @@
 import os
 import sys
 import rhsm.config
+import datetime
+
+
+class RhsmISO8601Formatter(logging.Formatter):
+    """Ensure date & time to be in ISO8601 format and containing info about milliseconds and time zone"""
+
```

Comment:
since this class represents a very specific format for the dates, I'd unconditionally create its parent class by passing `LOG_FORMAT` in all the cases: this way there is no need to manually build `RhsmISO8601Formatter(LOG_FORMAT)`